torch/_refs/__init__.py

```diff
@@ -2008,11 +2010,34 @@ def rot90(
         return clone(a)
 
 
-# update to cat then view instead of unsqueezing each tensor
+def _check_stack_inputs(tensors: TensorSequenceType) -> None:
```

Comment:
you don't need this check (and eager doesn't need it also ;-) ) because subsequent cat will verify that tensors are of the same size

Reply:
I guess this is probably still good to have here since it localizes the error to stack and not cat?

Reply:
I prefer not implementing error checks redundantly if possible. This would also require validating that tensors has one or more elements if it was kept